components/console-backend-service/internal/gqlschema/schema.graphql

```diff
@@ -706,8 +706,8 @@ type Query {
     pod(name: String!, namespace: String!): Pod
     pods(namespace: String!, first: Int, offset: Int): [Pod!]!
 
-    configMap(name: String!, namespace: String!): ConfigMap
-    configMaps(namespace: String!, first: Int, offset: Int): [ConfigMap!]!
+    configMap(name: String!, namespace: String!): ConfigMap @HasAccess(attributes: {resource: "configmap", verb: "get", apiGroup: "", apiVersion: "v1", namespaceArg: "namespace"})
```

Comment:
```suggestion
    configMap(name: String!, namespace: String!): ConfigMap @HasAccess(attributes: {resource: "configmaps", verb: "get", apiGroup: "", apiVersion: "v1", nameArg: "name", namespaceArg: "namespace"})
```
We always use plural name for resource name, so here it would also be `configmaps`. Also, for `get`, `update` and `delete` operations we want to grab resource name from provided attributes when it's possible, please add `nameArg` to update and delete mutations as well.